packages/kokkos-kernels/src/sparse/KokkosSparse_gauss_seidel_handle.hpp

```diff
@@ -50,292 +50,292 @@
+    bool called_symbolic;
+    bool called_numeric;
 
 
-  scalar_persistent_work_view_t permuted_y_vector;
-  scalar_persistent_work_view_t permuted_x_vector;
+    scalar_persistent_work_view_t permuted_y_vector;
+    scalar_persistent_work_view_t permuted_x_vector;
 
-  int suggested_vector_size;
-  int suggested_team_size;
+    int suggested_vector_size;
+    int suggested_team_size;
 
-  scalar_persistent_work_view_t permuted_diagonals;
-  nnz_lno_t block_size; //this is for block sgs
+    scalar_persistent_work_view_t permuted_inverse_diagonal;
+    nnz_lno_t block_size; //this is for block sgs
 
-  nnz_lno_t max_nnz_input_row, num_values_in_l1, num_values_in_l2, num_big_rows;
-  size_t level_1_mem, level_2_mem;
+    nnz_lno_t max_nnz_input_row, num_values_in_l1, num_values_in_l2, num_big_rows;
+    size_t level_1_mem, level_2_mem;
   public:
 
-  /**
-   * \brief Default constructor.
-   */
-  GaussSeidelHandle(GSAlgorithm gs = GS_DEFAULT):
-    owner_of_coloring(false),
-    algorithm_type(gs),
-    color_set_xadj(), color_sets(), numColors(0),
-    permuted_xadj(),  permuted_adj(), permuted_adj_vals(), old_to_new_map(),
-    called_symbolic(false), called_numeric(false), permuted_y_vector(), permuted_x_vector(),
-    suggested_vector_size(0), suggested_team_size(0), permuted_diagonals(), block_size(1), max_nnz_input_row(-1),
-	num_values_in_l1(-1), num_values_in_l2(-1),num_big_rows(0), level_1_mem(0), level_2_mem(0)
+    /**
+     * \brief Default constructor.
+     */
+    GaussSeidelHandle(GSAlgorithm gs = GS_DEFAULT):
+      owner_of_coloring(false),
+      algorithm_type(gs),
+      color_set_xadj(), color_sets(), numColors(0),
+      permuted_xadj(),  permuted_adj(), permuted_adj_vals(), old_to_new_map(),
+      called_symbolic(false), called_numeric(false), permuted_y_vector(), permuted_x_vector(),
+      suggested_vector_size(0), suggested_team_size(0), permuted_inverse_diagonal(), block_size(1), max_nnz_input_row(-1),
+      num_values_in_l1(-1), num_values_in_l2(-1),num_big_rows(0), level_1_mem(0), level_2_mem(0)
     {
-    if (gs == GS_DEFAULT){
-      this->choose_default_algorithm();
-    }
+      if (gs == GS_DEFAULT){
+        this->choose_default_algorithm();
+      }
 
 
-  }
+    }
 
-  void set_block_size(nnz_lno_t bs){this->block_size = bs; }
-  nnz_lno_t get_block_size(){return this->block_size;}
+    void set_block_size(nnz_lno_t bs){this->block_size = bs; }
+    nnz_lno_t get_block_size(){return this->block_size;}
 
     /** \brief Chooses best algorithm based on the execution space. COLORING_EB if cuda, COLORING_VB otherwise.
-   */
-  void choose_default_algorithm(){
+     */
+    void choose_default_algorithm(){
 #if defined( KOKKOS_ENABLE_SERIAL )
-    if (Kokkos::Impl::is_same< Kokkos::Serial , ExecutionSpace >::value){
-      this->algorithm_type = GS_PERMUTED;
+      if (Kokkos::Impl::is_same< Kokkos::Serial , ExecutionSpace >::value){
+        this->algorithm_type = GS_PERMUTED;
 #ifdef VERBOSE
```

Comment:
It's a bad idea to have macros with no namespacing (e.g., `KOKKOSSPARSE_VERBOSE` would be nice), but again, that's not your fault.